File: projekat/src/app/entities/airline/airline.ts
import { Company } from '../company/company';
import { Location } from '../location/location';

export class Airline extends Company {
    location: Location;
    //destinations:
    //flights:
    //discountedTickets:
    //configuration?
    //pricelist

    constructor(name: string, description: string, logo: string) {
        super(name, description, logo);
    }
}

File: BookingService/BookingAPI/Model/Airline.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace BookingAPI.Model
{
    public class Airline
    {
        [Key]
        public string Name { get; set; }
        public string Description { get; set; }
        public string AdminUsername { get; set; }
        public Image Logo { get; set; }
        //public List<City> Destinations { get; set; }
        //public List<Flight> Flights { get; set; }
        // Spisak karata sa popustima za brzu rezervaciju
        // Konfiguracija segmenta i mesta u avionima
        // Cenovnik i informacije o prtljagu
    }
}
